code/game/objects/items/devices/motion_detector.dm

```diff
@@ -81,23 +90,27 @@
 
 	playsound(loc, 'sound/items/detector.ogg', 60, 0, 7, 2)
 
-	var/mob/living/carbon/human/human_user
-	if(ishuman(loc))
-		human_user = loc
-
 	var/detected
-	for(var/mob/living/M in orange(detector_range, src))
-		if(M == loc) continue //device user isn't detected
-		if(!isturf(M.loc)) continue
-		if(world.time > M.l_move_time + 20) continue //hasn't moved recently
-		if(isrobot(M)) continue
+	for(var/mob/living/M in orange(detector_range, human_user))
+		to_chat(world, "<span class='danger'>MOTION SENSOR DEBUG: Moving mob detected: [M.name]. Detector range: [detector_range].</span>")
```

Comment:
Please don't push debug messages to PRs, especially ones that are being testmerged.